File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/OcsEmr/OCS2015U06EmrTagInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.OcsEmr
{
    [Serializable]
	public class OCS2015U06EmrTagInfo
	{
		private String _tagId;
		private String _tagCode;
		private String _tagName;
		private String _tagLevel;
		private String _tagParent;
		private String _description;
		private String _userId;
		private String _filterFlg;
		private String _created;
		private String _updated;

		public String TagId
		{
			get { return this._tagId; }
			set { this._tagId = value; }
		}

		public String TagCode
		{
			get { return this._tagCode; }
			set { this._tagCode = value; }
		}

		public String TagName
		{
			get { return this._tagName; }
			set { this._tagName = value; }
		}

		public String TagLevel
		{
			get { return this._tagLevel; }
			set { this._tagLevel = value; }
		}

		public String TagParent
		{
			get { return this._tagParent; }
			set { this._tagParent = value; }
		}

		public String Description
		{
			get { return this._description; }
			set { this._description = value; }
		}

		public String UserId
		{
			get { return this._userId; }
			set { this._userId = value; }
		}

		public String FilterFlg
		{
			get { return this._filterFlg; }
			set { this._filterFlg = value; }
		}

		public String Created
		{
			get { return this._created; }
			set { this._created = value; }
		}

		public String Updated
		{
			get { return this._updated; }
			set { this._updated = value; }
		}

		public OCS2015U06EmrTagInfo() { }

		public OCS2015U06EmrTagInfo(String tagId, String tagCode, String tagName, String tagLevel, String tagParent, String description, String userId, String filterFlg, String created, String updated)
		{
			this._tagId = tagId;
			this._tagCode = tagCode;
			this._tagName = tagName;
			this._tagLevel = tagLevel;
			this._tagParent = tagParent;
			this._description = description;
			this._userId = userId;
			this._filterFlg = filterFlg;
			this._created = created;
			this._updated = updated;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/emr/OCS2015U06EmrTagInfo.java
package nta.med.data.model.ihis.emr;

import java.math.BigDecimal;
import java.util.Date;

public class OCS2015U06EmrTagInfo {
    private Integer tagId;
    private String tagCode;
    private String tagName;
    private String tagLevel;
    private String tagParent;
    private String description;
    private String hospCode;
    private String userId;
    private BigDecimal filterFlg;
    private BigDecimal activeFlg;
    private Date created; 
    private Date updated;
    
	public OCS2015U06EmrTagInfo(Integer tagId, String tagCode, String tagName,
			String tagLevel, String tagParent, String description,
			String hospCode, String userId, BigDecimal filterFlg,
			BigDecimal activeFlg, Date created, Date updated) {
		super();
		this.tagId = tagId;
		this.tagCode = tagCode;
		this.tagName = tagName;
		this.tagLevel = tagLevel;
		this.tagParent = tagParent;
		this.description = description;
		this.hospCode = hospCode;
		this.userId = userId;
		this.filterFlg = filterFlg;
		this.activeFlg = activeFlg;
		this.created = created;
		this.updated = updated;
	}
	public Integer getTagId() {
		return tagId;
	}
	public void setTagId(Integer tagId) {
		this.tagId = tagId;
	}
	public String getTagCode() {
		return tagCode;
	}
	public void setTagCode(String tagCode) {
		this.tagCode = tagCode;
	}
	public String getTagName() {
		return tagName;
	}
	public void setTagName(String tagName) {
		this.tagName = tagName;
	}
	public String getTagLevel() {
		return tagLevel;
	}
	public void setTagLevel(String tagLevel) {
		this.tagLevel = tagLevel;
	}
	public String getTagParent() {
		return tagParent;
	}
	public void setTagParent(String tagParent) {
		this.tagParent = tagParent;
	}
	public String getDescription() {
		return description;
	}
	public void setDescription(String description) {
		this.description = description;
	}
	public String getHospCode() {
		return hospCode;
	}
	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}
	public String getUserId() {
		return userId;
	}
	public void setUserId(String userId) {
		this.userId = userId;
	}
	public BigDecimal getFilterFlg() {
		return filterFlg;
	}
	public void setFilterFlg(BigDecimal filterFlg) {
		this.filterFlg = filterFlg;
	}
	public BigDecimal getActiveFlg() {
		return activeFlg;
	}
	public void setActiveFlg(BigDecimal activeFlg) {
		this.activeFlg = activeFlg;
	}
	public Date getCreated() {
		return created;
	}
	public void setCreated(Date created) {
		this.created = created;
	}
	public Date getUpdated() {
		return updated;
	}
	public void setUpdated(Date updated) {
		this.updated = updated;
	}
    
}
